plugins/janus_sip.c

```diff
@@ -4513,6 +4595,9 @@ static void janus_sip_media_cleanup(janus_sip_session *session) {
 	}
 	/* Clean up SRTP stuff, if needed */
 	janus_sip_srtp_cleanup(session);
+
+	/* Media fields not cleaned up elsewhere */
+	janus_sip_media_reset(session);
```

Comment:
Any reason why this is a separate function, if it's only called here?

Reply:
For all three plugins (writing it just here for the sake of brevity), it may be simpler to create a helper method that includes all that stuff, e.g., `janus_sipre_media_reset` or something like that. Doing this all inline looks ugly.

_Originally posted by @lminiero in https://github.com/_render_node/MDIzOlB1bGxSZXF1ZXN0UmV2aWV3VGhyZWFkMTkyMDU0MDQ0OnYy/pull_request_review_threads/discussion_